anvil-python/anvil/commands/haproxy.py

```diff
@@ -100,34 +142,55 @@ def has_prompts(self) -> bool:
         return True
 
     def prompt(self, console: Console | None = None) -> None:
-        variables = questions.load_answers(
-            self.client, self._KEEPALIVED_CONFIG
-        )
+        variables = questions.load_answers(self.client, self._HAPROXY_CONFIG)
         variables.setdefault("virtual_ip", "")
+        variables.setdefault("charm_haproxy_config", {})
 
         # Set defaults
         self.preseed.setdefault("virtual_ip", "")
+        self.preseed.setdefault("charm_haproxy_config", {})
 
-        keepalived_config_bank = questions.QuestionBank(
-            questions=keepalived_questions(),
+        haproxy_config_bank = questions.QuestionBank(
+            questions=haproxy_questions(),
             console=console,
             preseed=self.preseed.get("haproxy"),
             previous_answers=variables,
             accept_defaults=self.accept_defaults,
         )
 
-        variables["virtual_ip"] = keepalived_config_bank.virtual_ip.ask()
+        variables["virtual_ip"] = haproxy_config_bank.virtual_ip.ask()
+
+        cert_filepath = haproxy_config_bank.ssl_cert.ask()
```

Comment:
let's move this to Terraform. I like the idea better.

Reply:
omitting since we aren't actually passing the file contents as a string, but just a file path with both the cert and key concatenated